test/controllers/EmailControllerSpec.scala

```diff
@@ -17,11 +17,12 @@
 package controllers
 
 import base.BaseSpec
-
 import mocks.MockEmailService
 import play.api.libs.json.Json
 import play.api.test.FakeRequest
 import play.api.http.Status._
+import play.api.mvc.ControllerComponents
+import play.api.test.Helpers.stubControllerComponents
```

Comment:
Can you also remove these two `ControllerComponents` imports from the four affected controller specs now that the `val` has been moved to the base trait